.github/workflows/gen-openapi.yml

```diff
@@ -0,0 +1,27 @@
+name: API
+
+on: [push]
```

Comment:
Same here. It is better to run this within Pull Requests only and on the push to `main`. Example: https://github.com/qoollo/bob/blob/master/.github/workflows/build.yml#L3